manifests/charts/gateways/istio-ingress/templates/meshexpansion.yaml

```diff
@@ -37,6 +46,15 @@ spec:
         host: istiod.{{ .Values.global.istioNamespace }}.svc.{{ .Values.global.proxy.clusterDomain }}
         port:
           number: 15012
+  {{- if or .Values.sidecarInjectorWebhook.injectionURL .Values.base.validationURL }}
```

Comment:
Consistency - can use use Values.base.injectionURL ?

Reply:
the prob is the mutatingwebhook config is part as istio-discover/istiod while the validation webhook config is part of the base